src/styles/theme-dark.scss

```
@@ -257,12 +291,18 @@ page {
   --nutui-font-size-xs: var(--nutui-font-size-11);
   // 汉字：次要内容、辅助类信息 数字：羊角符号、小数点及角分
   --nutui-font-size-s: var(--nutui-font-size-12);
+  --nutui-font-size-m: var(--nutui-font-size-13);
   // 标准内容、名称类关键信息
```

Stylelint (17.11.1)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)